styles.css

```diff
@@ -357,10 +581,31 @@ body {
 }
 
 #directorHeading {
-color: #cccccc
+  color: #cccccc
 }
 
+.servicesIconImages {
+  height: 3.5rem;
+  width: 3.5rem;
+  text-align: center;
+}
+
+#servicesHeading {
+  color: #f2f2f2;
+}
+
+#historyArrow {
+  border-left: .23rem solid transparent;
+  border-right: .23rem solid transparent;
+  border-top: .7rem solid #e6e6e6;
+  margin: 0rem 0rem 0rem 0rem;
```

Comment:
since all of them are set to the same value, we can use a shorthand here `margin: 0`